### ISSUE TITLE
API Middleware Is Lost When Deployed To Apache

### ISSUE THREAD

**Issue by LouieLightning** [react-boilerplate/react-boilerplate#2769]
## Description
I'm using Apache on Centos 7 to deploy my project. After running npm run build and moving the files to the Apache server the application does load up and serves routes. However when a container tries to make an API call it return html which is correcting because of the rewrite Apache does.. I altered the htaccess file to ignore api calls but now it says my route is not found. This works fine in a non-built environment. Do I need a reverse proxy server for node/express or should I deploy express separately? Or does react-router handle api calls? I cannot find much documentation about this.

**Expected behavior**
I expect API endpoint to be called and return proper response.

## Versions

- React-Boilerplate: 4.0.0
- Node: 10.15.1
- NPM: 6.7.0
- Browser: FireFox 69.0.2 & Chrome 77.0.3865.90

**Comment by julienben** [comment]
If you need the node/express server to proxy requests to your backend then you have no choice but to deploy to a node server, not Apache...

Or maybe you can somehow config Apache to do the proxying for you on a certain set of routes.

The simplest solution though is just to point your frontend to your backend directly. You'll just have to configure CORS on your backend.

**Comment by LouieLightning** [comment]
Julien, thank you for the reply even thought my issue was auto closed. 
The solution I came up with was to deploy the node/express server. Then I configured Apache as a reverse proxy server to send all requests to the node/express server. I added a condition to htaccess to ignore any calls made to the route /api/. 
This setup works well for us. I work at a small company where it’s just my manager and I in IT. He handles all the ops side of devOps and I handle the dev side.
